glados.py

```diff
@@ -282,14 +282,8 @@ def _process_detected_audio(self):
         if detected_text:
             logger.success(f"ASR text: '{detected_text}'")
 
-            if self.wake_word is not None:
-                if self._wakeword_detected(detected_text):
-                    logger.info("Wake word detected!")
-
-                    self.llm_queue.put(detected_text)
-                    self.processing = True
-                else:
-                    logger.info("No wake word detected. Ignoring...")
+            if self.wake_word and not self._wakeword_detected(detected_text):
+                logger.info("Required wake word not detected.")
```

Comment:
You could make such logs more informative almost without effort - I took the practice to always include all var values that cause the message to be sent.

Here it would be:

```suggestion
                logger.info(f"Required wake word {self.wake_word=} not detected.")
```